javascript/index.js

```diff
@@ -14,52 +14,94 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+  printMinutes();
+  printSeconds();
+  printMilliseconds();
 }
 
 function printMinutes() {
-  // ... your code goes here
+  minDec.innerText = chronometer.getMinutes()[0];
+  minUni.innerText = chronometer.getMinutes()[1];
```

Comment:
Invoking the function twice might not be good performance wise, as they carry the same value, it would be a better approach to invoke que function once, save it inside a variable and then apply the different digits to the different DOM elements.